association_test.go

```diff
@@ -224,3 +225,1031 @@ func (c *dumbConn) SetReadDeadline(t time.Time) error {
 func (c *dumbConn) SetWriteDeadline(t time.Time) error {
 	return nil
 }
+
+////////////////////////////////////////////////////////////////////////////////
+// testConn, connBridge for emulating dtls.Conn
+
+type testConn struct {
```

Comment:
Maybe we should conciser moving this to https://github.com/pions/transport ?

Reply:
Agreed.

Reply:
done @f8bfd